tests/test_configuration.py

```diff
@@ -799,6 +824,24 @@ def setUpClass(cls):
 	def tearDownClass(cls):
 		sys.path.pop(0)
 
+class TestBar(TestCase):
+	def test_bar(self):
+		import powerline as powerline_module
+		with swap_attributes(config, powerline_module):
+			with get_powerline_raw(config, powerline_module, ext='wm', renderer_module='bar') as powerline:
+				self.assertEqual("%{{l}}{}%{{r}}{}".format(powerline.render(side='left'), powerline.render(side='right')), "SOME_STRING_HERE")
```

Comment:
Ah, no. If you have some plans about disabling left or right parts it is better to leave everything as-is, *except* that you must not use `%{l}` and `%{r}` in tests. If you have no plans it is better to move it to `bar.py` like Vim renderer in `powerline/renderers/vim.py`.